CAN_Logger_with_Autobaud_and_Requests_no_CAN2_AES_CBC_SHA256/CAN_Logger_with_Autobaud_and_Requests_no_CAN2_AES_CBC_SHA256.ino

```diff
@@ -678,6 +655,7 @@ void open_binFile(){
   else
   {
     YELLOW_LED_fast_blink_state == false;
```

Comment:
"YELLOW_LED_fast_blink_state == false;" 
is boolean, 
should it be:
"if (YELLOW_LED_fast_blink_state == false) write_initial_meta_data()"; 
or
"YELLOW_LED_fast_blink_state = false;"
?